src/Microsoft.Sbom.Adapters/Adapters/ComponentDetection/SpdxPackageComponentExtensions.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.ComponentDetection.Contracts.TypedComponent;
+using Microsoft.Sbom.Contracts;
+
+namespace Microsoft.Sbom.Adapters.ComponentDetection;
+
+internal static class SpdxPackageComponentExtensions
+{
+    public static SbomPackage? ToSbomPackage(this SpdxPackageComponent spdxPackageComponent, string? license = null) => new()
```

Comment:
This class can be removed as well.

Reply:
This class is a duplicate of https://github.com/microsoft/sbom-tool/blob/main/src/Microsoft.Sbom.Adapters/Adapters/ComponentDetection/SpdxComponentExtensions.cs

Reply:
This class is actually not duplicate of SpdxComponentExtensions.

SpdxComponentExtensions focus on displaying the information about external spdx document. it has few specific properties like documentnamespace etc.

wherease SpdxPackageComponentExtensions is display package information within the document itself.

Reply:
I see it now. Thank you for clarifying. [(.](https://github.com/microsoft/sbom-tool/blob/main/src/Microsoft.Sbom.Api/Executors/PackagesWalker.cs#L30) We don't need detailed information about the SPDX Documents detected. We just list them as an external reference.

Reply:
I tested with commenting the ` .Where(component => !(component.Component is SpdxComponent)) ` from packageswalker and found that it does not include packages info from projectA .. I am missing something more here. Please guide